frontend/src/components/common/Logo/Logo.tsx

```diff
@@ -0,0 +1,23 @@
+import { useContext } from 'react';
+import { FC } from 'react';
+import logoDarkTheme from '../../../assets/logo.svg';
+import logoLightTheme from '../../../assets/logoLightTheme.svg';
+import { ThemeContext } from '../../../contexts/ThemeContext';
+
+export interface ILogoProps {
+  width: number;
+}
+
+const Logo: FC<ILogoProps> = ({ ...props }) => {
+  const { isDarkTheme } = useContext(ThemeContext);
+  const { width } = props;
+  return (
+    <img
+      src={isDarkTheme ? logoDarkTheme : logoLightTheme}
```

Comment:
Since the difference between the two logos seems to be only related to color, could it be changed dynamically through CSS instead of having two versions?

Reply:
I tried to change it using CSS, but without success. Maybe would be better to discuss about it during the meeting. According to the react documentation https://it.reactjs.org/docs/context.html#when-to-use-context is a good idea to use a context to choose the used theme. Maybe i could be try to hardcode the svg in the logo component to avoid file duplication (i'm talking about logo.svg and logoLightTheme.svg)